pro/src/pages/AdageIframe/app/components/OffersInstantSearch/OffersSearch/Offers/Offer.tsx

```diff
@@ -46,8 +47,9 @@ const Offer = ({
   position,
   afterFavoriteChange,
   isInSuggestions,
+  openDetails,
```

Comment:
```suggestion
  openDetails = false,
```
Comme ça on peut supprimer ?? en dessous

Reply:
ah oui bien vu